test/client/spec/victory-shared-events/victory-shared-events.spec.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Client tests
+ */
+import { curry, flow, forEach } from "lodash";
+import { mount } from "enzyme";
+import React from "react";
+import { addEvents } from "src/index";
+import VictorySharedEvents from "src/victory-shared-events/victory-shared-events";
+import { MockVictoryComponent, MockDataComponent } from "../mock-components";
+
+describe("components/victory-shared-events", () => {
+  const EventedMockVictoryComponent = addEvents(MockVictoryComponent);
+
+  const findVictoryComponentByName = curry((name, component) => {
```

Comment:
Prop selectors actually don't work right now for `mount`ed custom components - only on standard DOM components or custom components rendered with `shallow`. Let me see if shallow rendering would still work here - if it does then I can switch it out, but `find(/filter)Where` is currently a usable workaround for this issue on `mount`ed components. 

https://github.com/airbnb/enzyme/issues/534